boards/arduino-mkr1000/Makefile.dep

```diff
@@ -1,6 +1,6 @@
 USEMODULE += boards_common_arduino-mkr
 
-ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
+ifneq (,$(filter gnrc_netdev_default netdev_default,$(USEMODULE)))
```

Comment:
You can drop `gnrc_netdev_default`

Reply:
Are you sure, does `gnrc_netdev_default` depend on `netdev_default`

Reply:
Changed and squashed it directly.